prac_08/convert_miles_km.kv

```diff
@@ -0,0 +1,25 @@
+BoxLayout:
+    orientation: 'vertical'
+    BoxLayout:
+        orientation: 'horizontal'
+        TextInput:
+            id: input_miles
+            font_size: 48
+            on_text: app.handle_calculate(self.text)
+        BoxLayout:
+            orientation: 'vertical'
+            size_hint_x: 0.25
+            Button:
+                text: 'Up'
+                on_press: app.handle_increment(input_miles.text, 1)
```

Comment:
Does the `handle_increment` method have a parameter for `input_miles.text`?
As above, you don't have to pass in `input_miles.text` if you don't want to.